components/View/style.js

```diff
@@ -37,16 +34,16 @@ const content = (
     top: isFullscreen ? 0 : `calc(${navHeight}px + var(--safe-area-inset-top))`,
     display: 'flex',
     flexDirection: 'column',
-    paddingBottom: 'calc(var(--tabbar-height) + var(--safe-area-inset-bottom))',
     bottom: 0,
-    ...considerPaddingTop && {
-      marginBottom: 'calc(var(--tabbar-height) + var(--safe-area-inset-bottom))',
-      bottom: -24,
-    },
   }).toString();
 };
 
+const padding = css({
```

Comment:
I've found some issues that seem to be related to your changes. Additionally i think that your changed also need to be applied to the GMD theme since this could also be used on iOS.

![img_0125](https://user-images.githubusercontent.com/13087792/43009548-497497aa-8c3e-11e8-949b-2b9e6897f958.PNG)
![img_0123](https://user-images.githubusercontent.com/13087792/43009550-498fd86c-8c3e-11e8-9941-c7ac6ffd0ff0.PNG)
![img_0121](https://user-images.githubusercontent.com/13087792/43009551-49a5d040-8c3e-11e8-9111-aee06474e21f.PNG)
![img_0128](https://user-images.githubusercontent.com/13087792/43009552-49c0f74e-8c3e-11e8-9f65-09e90199c665.PNG)
![img_0127](https://user-images.githubusercontent.com/13087792/43009553-49dc9422-8c3e-11e8-9699-4258064e87dd.PNG)
![img_0126](https://user-images.githubusercontent.com/13087792/43009554-49fa6b0a-8c3e-11e8-802a-d93182099843.PNG)
![img_0122](https://user-images.githubusercontent.com/13087792/43009556-4a16f0b8-8c3e-11e8-8f91-e0f7546cb443.PNG)
![img_0124](https://user-images.githubusercontent.com/13087792/43009557-4a33ad66-8c3e-11e8-975d-af032043ea18.PNG)